md2cf/replacements.py

```diff
@@ -0,0 +1,70 @@
+import importlib.util
+import json
+import re
+from typing import List
+
+from md2cf.console_output import console
+
+
+class Replacement:
+    def __init__(
+        self, name: str, pattern: str, new_value: str, evaluate: bool = False
+    ) -> None:
+        self.name = name
+        self.pattern = pattern
+        self.new_value = new_value
+        self.evaluate = evaluate
+
+    def replace(self, page):
+        console.print(f"Performing replacement '{self.name}'")
```

Comment:
Don't add console logging inside library classes/functions, please. This is better handled in `__main__.py`, but I'd drop it altogether since it breaks the TUI paradigm of the rest of the tool.